EnvironmentSetup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Setup script which install the Space Concordia Robotics Software team's development environment. 
+
+# Setup venv
+virtualenv -p `which python3.6` venv
+source venv/bin/activate
+
+# Install Requirements
+pip install -r requirements.txt -r requirements-dev.txt
+
+# Setup python and test it
+python setup.py develop
+
+# Install ROS-Kinetic
+wget https://raw.githubusercontent.com/ROBOTIS-GIT/robotis_tools/master/install_ros_kinetic.sh
+chmod 755 ./install_ros_kinetic.sh
+bash ./install_ros_kinetic.sh
+
+sudo apt install ros-kinetic-rosbridge-suite
+
+# Install camera stuff
+sudo apt-get install ros-kinetic-cv-camera
+sudo apt-get install ros-kinetic-web-video-server
+
+# Build catkin
+cd ~/Programming/robotics-prototype/robot/rospackages
+catkin_make
```

Comment:
I just setup one of the space lab computers with the environment so I know for a fact just running `catkin_make` should throw some errors and not complete the build properly.
We need to make sure to run `rosdep install --from-paths src/ --ignore-src -r -y` on that `src` folder _before_ running `catkin_make`. I think it will automatically install apt packages like `ffmpeg`. Please investigate/confirm this. If it covers all the bases we can remove all the `apt-get install` statements.

Reply:
couldn't find out if it install packages

Reply:
rosdep install should install all apt packages used by the ros packages